src/protocol.rs

```diff
@@ -5,7 +5,7 @@ pub struct LineProtocol(String);
 /// Used to convert Rust types to influx types. Must be
 /// implemented by any type that will be used as a Field in a [crate::Point].
 pub trait IntoFieldData {
-    fn into_field_data(&self) -> FieldData;
```

Comment:
Rationale for renaming this? Don't think I'm a big fan of it since that's a breaking change for anybody using this trait.

Reply:
Just because `clippy` will generate a warning, suggesting that the methods of the into trait are named after into

Reply:
fair enough, I would like to adhere to clippy. 👍